src/components/StyleGuide/ColorCard.tsx

```diff
@@ -6,41 +6,57 @@ import Icon from "../Icons/Icon";
 import SimpleGrid from "../Grid/SimpleGrid";
 import Table from "../Table/Table";
 import Text from "../Text/Text";
+import useNYPLTheme from "../../hooks/useNYPLTheme";
 
-export interface ColorCardProps {
-  /** The backgroundColor of the color card. */
-  backgroundColor: string;
-  /** The name of a color's javascript theme object. */
-  colorName: string;
-  /** The name of the color that the current color is based on. */
-  colorSource: string;
+export interface DataTableProps {
+  /** Contrast and WCAG compliance data related to the color gray.xxxx-dark when
+   * used with the current color. */
+  dataBgPageColor: string[];
+  /** Contrast and WCAG compliance data related to the color gray.xxx-dark when
+   * used with the current color. */
+  dataBgDefaultColor: string[];
   /** Contrast and WCAG compliance data related to the color black when used
    * with the current color. */
-  dataBlack: string[];
+  dataBlackColor: string[];
+  /** Contrast and WCAG compliance data related to the default color for
+   * heading elements when used with the current color. */
+  dataHeadingColor?: string[];
+  /** Contrast and WCAG compliance data related to the default color for basic
+   * text elements when used with the current color. */
+  dataTextColor?: string[];
   /** Contrast and WCAG compliance data related to the color white when used
    * with the current color. */
-  dataWhite: string[];
+  dataWhiteColor: string[];
   /** The color to use for text in the color card. */
   textColor: string;
 }
 
-export interface DataTableProps {
-  /** Contrast and WCAG compliance data related to the color black when used
-   * with the current color. */
-  dataBlack: string[];
-  /** Contrast and WCAG compliance data related to the color white when used
-   * with the current color. */
-  dataWhite: string[];
-  /** color to use for text in color card */
-  textColor: string;
+export interface ColorCardProps extends DataTableProps {
+  /** The backgroundColor of the color card. */
+  backgroundColor: string;
+  /** The name of a color's javascript theme object. */
+  colorName: string;
+  /** The name of the color that the current color is based on. */
+  colorSource: string;
 }
 
 export const DataTable = (props: React.PropsWithChildren<DataTableProps>) => {
   const {
-    dataBlack = ["--", "--", "--"],
-    dataWhite = ["--", "--", "--"],
+    dataBgPageColor,
+    dataBgDefaultColor,
+    dataBlackColor = ["--", "--", "--"],
```

Comment:
These are okay because all color cards should, at the minimum, have the black and white color rows. The other are optional, so the default is okay for the two variables noted.